nss-c31-unusual-prestige/nutshell
# Cm news session

# Description
When logging in the news articles that have the current users matching ID will be displayed along with any added news articles.

Applies to issue # 7

## Type of change

Please delete options that are not relevant.




# Testing Instructions

Pull down branch and run grunt
Put this in main.js
newsForm.newsForm()
newsFeedDOM.childArticleContainer()
newsFeedDOM.listAllNews()


import homepage from "./Homepage/homePage.js"
import messageHandler from "./chat/messages.js"


homepage.homePage()
messageHandler.createChat();

after logging in only the news articles with the matching user_id should appear on DOM along with articles that you add after the page is loaded.



# Checklist:
